pages/Javascript-API.md

```diff
@@ -56,13 +89,69 @@ The properties of the feature at the current cursor location may be accessed thr
 
 This is accomplished by assigning a unique color to each feature onscreen and rendering the scene to an offscreen buffer. When queried, the `scene.selection` object checks the offscreen render at the current cursor position, and identifies the feature by its color.
 
-#### `requestRedraw()`
-Requests an update to the drawn map. If `animated: true` is set, this happens once per frame automatically.
-
 #### `getActiveCamera()`
+Returns the active camera.
 
-#### `setActiveCamera()`
+#### `requestRedraw()`
+Requests an update to the drawn map. If `animated: true` is set, this happens once per frame automatically.
 
 #### `reload(scene_url)`
```

Comment:
Also, the scene URL is optional in `load()`, if not specified, the current URL will be reloaded.